app/src/test/java/io/gnosis/safe/ThreeTenTimezoneDatabaseInit.kt

```diff
@@ -0,0 +1,13 @@
+package io.gnosis.safe
+
+import org.threeten.bp.zone.TzdbZoneRulesProvider
+import org.threeten.bp.zone.ZoneRulesProvider
+
+fun Any.initThreeTen() {
```

Comment:
This is needed to run unit tests.

Reply:
is there any specific reason for having this as an extension for Any?

Reply:
The resource loader needs a class (to get the class loader).

Reply:
Since this is not needed with desugaring I will resolve this conversation.